<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerWindow</class>
 <widget class="QMainWindow" name="ControllerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Script</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="currentScript">
        <item>
         <property name="text">
          <string>ib 0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rs</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteLineButton">
        <property name="text">
         <string>Delete Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runScriptButton">
        <property name="text">
         <string>Run Script</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="resetCardButton">
        <property name="text">
         <string>Reset Card</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="readCurrentSectorButton">
        <property name="text">
         <string>Read Current Sector</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="resetBufferButton">
        <property name="text">
         <string>Reset Buffer</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="readCSDButton">
        <property name="text">
         <string>Read CSD</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="readCIDButton">
        <property name="text">
         <string>Read CID</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="setBufferValueText"/>
        </item>
        <item>
         <widget class="QPushButton" name="setBufferValueButton">
          <property name="text">
           <string>Set Buffer Value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="setBufferOffsetText"/>
        </item>
        <item>
         <widget class="QPushButton" name="setBufferOffsetButton">
          <property name="text">
           <string>Set Buffer Offset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="setSectorText"/>
        </item>
        <item>
         <widget class="QPushButton" name="setSectorButton">
          <property name="text">
           <string>Set Sector</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="writeSectorButton">
        <property name="text">
         <string>Write Current Sector</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextBrowser" name="networkConsole"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
